src/Diagnostics.RuntimeHost/Controllers/AppServiceDomain/AppServiceDomainController.cs

```diff
@@ -52,5 +52,11 @@ public async Task<IActionResult> GetInsights(string subscriptionId, string resou
         {
             return await base.GetInsights(GetResource(subscriptionId, resourceGroupName, domainName), supportTopicId, minimumSeverity, startTime, endTime, timeGrain);
         }
+
+        [HttpPost(UriElements.Publish)]
+        public async Task<IActionResult> PublishDetector(string subscriptionId, string resourceGroupName, string domainName, [FromBody] DetectorPackage pkg)
```

Comment:
Do we need a resource specific endpoint for this operation?

Reply:
Not Really. Just to stay consistent.